apis/spotify.py

```diff
@@ -42,19 +42,19 @@ def spotify_get(url: str) -> Dict:
     api_data = {}
 
     spotify_id = url.split("/")[-1]

```

Comment:
you can use urlparse

https://docs.python.org/3/library/urllib.parse.html

Reply:
Done! I think there's a couple more places we could use this.